ComfortZoneApp/app/src/main/res/layout/input_layout.xml

```diff
@@ -37,11 +44,11 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginBottom="8dp"
+        app:boxBackgroundColor="@color/grey"
```

Comment:
Do we need to add an id for this component TextInputLayout as well? Generally it's a good practice to add id to all components.